migrations/versions/1030_buyer_email_domain_data_migration.py

```diff
@@ -0,0 +1,47 @@
+"""Migrate buyer email domains from .txt file to DB
+
+Revision ID: 1030
+Revises: 1020
+Create Date: 2017-10-13 11:18:22.683693
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '1030'
+down_revision = '1020'
+
+buyer_domains_table = sa.Table(
+    'buyer_email_domains',
+    sa.MetaData(),
+    sa.Column('id', sa.Integer, primary_key=True),
+    sa.Column('domain_name', sa.String, nullable=True),
+)
+
+
+def upgrade():
+    try:
+        with open('./data/buyer-email-domains.txt') as f:
+            legacy_buyer_domains = [line.strip() for line in f]
+    except FileNotFoundError as e:
+        # When we eventually delete the file, we can exit the migration early
```

Comment:
"Eventually"?! We should do it right away.

Reply:
I promise to do another PR as soon as this one has deployed 😸